File: src/PiWol.WebApp/ClientApp/src/app/admin/hosts/models/host.model.ts
export class HostModel {
  id: string;
  macAddress: string;
  ipAddress: string;
  netmask: string;
  hostname: string;
  status: HostStatus;
}

export enum HostStatus {
  unknown = 0,
  online = 1,
  offline = 2
}

File: src/PiWol.WakeOnLanService.Abstraction/Models/HostModel.cs
using System;

namespace PiWol.WakeOnLanService.Abstraction.Models
{
    public class HostModel
    {
        public Guid Id { get; set; }

        public string Hostname { get; set; }

        public string IpAddress { get; set; }

        public string MacAddress { get; set; }

        public string Netmask { get; set; }

        public HostStatus Status { get; set; }
    }
}